pkg/rehearse/configmaps.go

```diff
@@ -144,23 +178,20 @@ func replaceSpecNames(namespace string, cfg prowplugins.ConfigUpdater, mapping m
 		}
 		if name, ok := mapping[v.Name]; ok {
 			v.Name = name
+			v.Namespaces = []string{""}
+			ret.Maps[k] = v
```

Comment:
This is one tricky change: this prevents us from building a config-updater config (that will ultimately be used to populate the temporary CMs) that updates the production CMs on bugs in the logic. Technically this should no longer happen because our logic is overall fixed, but better safe then sorry.